H2TauTau/cfgPython/tt/diTau_2018_modular_cfg.py

```diff
@@ -234,13 +235,42 @@ def sorting_metric(dilepton):
 
 # weights ================================================================
 
+# id weights
 from CMGTools.H2TauTau.heppy.analyzers.TauIDWeighter import TauIDWeighter
+tauidweighter_general = cfg.Analyzer(
+    TauIDWeighter,
+    'TauIDWeighter_general',
+    taus = lambda event: [event.dileptons_sorted[0].leg1(),event.dileptons_sorted[0].leg2()]
+)
+
 tauidweighter = cfg.Analyzer(
     TauIDWeighter,
     'TauIDWeighter',
-    taus = lambda event: [event.dileptons_sorted[0].leg1(),event.dileptons_sorted[0].leg2()]
+    taus = lambda event: [event.dileptons_sorted[0].leg1(),event.dileptons_sorted[0].leg2()],
+    WPs = {'JetToTau':'Tight', # dummy, no weights for jet fakes
```

Comment:
Is it channel dependant? If so, where to find this information for et and mt?

Reply:
Yes it is see https://twiki.cern.ch/twiki/bin/viewauth/CMS/HiggsToTauTauWorking2017#etau_checklist